modules.d/95fcoe-uefi/parse-uefifcoe.sh

```diff
@@ -10,6 +10,9 @@ print_fcoe_uefi_conf() {
     [ -z "$mac" ] && return 1
     dev=$(set_ifname fcoe "$mac")
     vlan=$(get_fcoe_boot_vlan "$1")
+    if [ -z "$vlan" ]; then
```

Comment:
Please move this into the case statement below, and simplify the `if` ... `case` section.